src/index.js

```diff
@@ -315,6 +315,20 @@ class Kronos extends Component {
     let input = e.target.value
     let datetime = Moment(input, this.format(), true)
     if (datetime.isValid()) {
+
+      // this changes is to create proper moment object with proper timezone.
+      // So that, it does not mess up the date to isoString convertion.
+      // In dropdown this.state.datetime is being used to calculate the time range,
+      // so we are using this moment object to create a new moment object for the
+      // choosen time/ date. {line no. : 324~333}
+      if (this.props.format === 'DD/MM') {
```

Comment:
@anindya-ghosh what about the year?

Reply:
Yes we should add that also. I fixed this only for the specific case.

Reply:
I think we should wait until that case arises.

Reply:
@anindya-ghosh as discussed, rider's DoB could be one case